src/cordova/setup.action.mjs

```diff
@@ -62,6 +62,12 @@ export async function main(...parameters) {
   if (isApple) {
     // since apple can only be build on darwin systems, we don't have to worry about windows support here
     // TODO(daniellacosse): move this to a cordova hook
+    execSync(`cd third_party/CocoaLumberjack && make clean && make`, {
```

Comment:
This will make the build take much longer. Remove the binary from version control and call `make` only for an incremental build.

Reply:
unfortunately we have to build when the user's swift version changes, i don't think make tracks that

Reply:
The Swift version only changes when we update XCode, but we can assume a fixed XCode version. If one updates XCode, they should do a full clean build.

Let's remove the `make clean` and delete the binaries. We shouldn't have both the binaries and the make call. It's either one or the other.

Reply:
Oh, we should remove the binaries on the package.json "clean" script as well. We do that on the server code: https://github.com/Jigsaw-Code/outline-server/blob/733fa52f5cbc2e922505ed6d573456523762009d/package.json#L30

Reply:
How should we notify users that they're responsible to run make clean when they update xcode (which happens when you upgrade your OS)?

Reply:
That's a valid question. Perhaps we write in the README. The build will fail, not silently, so at least they know they need to do something. 

I think that's a better experience than have to rebuild everything whenever we rebuild the platform, which is already a pain I've been trough many times.

A better solution would be to add the dependencies to the Apple build, and have the apple build take care of it somehow, which would give us a better incremental build, but that's too advanced.

Reply:
Actually - npm run clean should take care of it now. So just telling people in the 'apple_development' doc should be enough.

Reply:
I did have a question though - I thought we decided we didn't like `make` scripts? Why not make this an `action`?